### PR TITLE
feat(unlock): Add sentry to unlock-app

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

Adds sentry to unlock-app and fixes the winston issue so we can upgrade to v7.

CleanShot 2022-11-23 at 19 24 24@2x

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/app.ts [hunk 1]
```diff
@@ -35,11 +35,7 @@ app.use(cookieParser())
 // Request logging
 app.use(
   expressWinston.logger({
-    transports: [
-      new winston.transports.Console({
-        silent: process.env?.NODE_ENV === 'test',
-      }),
-    ],
+    transports: [new winston.transports.Console({})],
```
Comment (julien51) [inline locksmith/src/app.ts:38]: We kind of want to silence in tests though?

File: unlock-app/src/pages/_error.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * See:
+ *  - https://nextjs.org/docs/basic-features/data-fetching/overview
+ *  - https://nextjs.org/docs/api-reference/data-fetching/get-initial-props
+ *  - https://reactjs.org/docs/error-boundaries.html
+ */
+
+import * as Sentry from '@sentry/nextjs'
+import NextErrorComponent from 'next/error'
+
+const CustomErrorComponent = (props) => {
```
Comment (julien51) [inline unlock-app/src/pages/_error.jsx:22]: Do we want to provide a little bit of UI here and point people to our discord for example?

**Review by julien51 — APPROVED** [review]

File: unlock-app/src/pages/_error.jsx (hunk 2 above)
Comment (searchableguy) [reply unlock-app/src/pages/_error.jsx:22]: Will do. That's a great idea.

File: locksmith/src/app.ts (hunk 1 above)
Comment (searchableguy) [reply locksmith/src/app.ts:38]: They are helpful in figuring out why a test isn't working in the CI hence removed.

**Review by julien51 — APPROVED** [review]
LGTM!

File: locksmith/src/app.ts (hunk 1 above)
Comment (julien51) [reply locksmith/src/app.ts]: ok

**Comment by julien51** [comment]
@searchableguy We're missing some credentials for Sentry here?

**Pull request merged**